### DISCUSSION TITLE
Saving Bookmarks with Time Signature to File (either .txt or .pdf)

### DISCUSSION THREAD

**Discussion by theGuruWithin** [PaulWoitaschek/Voice#2549] [Ideas]
Love Voice to read my audiobooks... really love the Bookmark Feature.  

Please consider adding the ability to print the Bookmarks from an Audiobook to a File (either .txt or .pdf).  This would be great.  

Also the ability to export or import the created file would also be nice....  Also would love to see a desktop version of Voice to use so one could go back and forth from Android Device to Win or Linux.

Thank you for all your work on this simple non invasive Audiobook player!

**Comment by UnexceptedSpectic** [comment]
In the meantime, @PaulWoitaschek, is there any way to manually do this for those whose devices aren't rooted?

**Comment by luzpaz** [comment]
Soft bump @PaulWoitaschek

**Comment by luzpaz** [comment]
This is especially important since there is an existing bug that if you rename the audio file, all bookmarks will get wiped:
* #3221

**Comment by camdencarr84-jpg** [comment]
PLEASE 🙏

**Comment by luzpaz** [comment]
Related:  
* #3221 
* #2636
